src/main/java/xyz/vegaone/easytrackingv3/dto/Project.java

```diff
@@ -0,0 +1,28 @@
+package xyz.vegaone.easytrackingv3.dto;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import xyz.vegaone.easytrackingv3.domain.SprintEntity;
+import xyz.vegaone.easytrackingv3.domain.TaskEntity;
+import xyz.vegaone.easytrackingv3.domain.UserEntity;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonIdentityInfo(generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Project {
+
+    private Long id;
+    private String projectName;
+    private String projectDescription;
+    private List<SprintEntity> sprints;
+    private List<TaskEntity> tasks;
+    private List<UserEntity> users;
```

Comment:
I checked out the branch locally and noticed that the 3 types should be their DTO equivalent. Could you please check and give it one more try? Let's see if this fixes your serialisation issue.

Could you also check the changeset related to the `project_description` and rename the column name to `description`? Pretty please:)

Reply:
That seems to solve the issue! Great job!